pkg/controller/core/workload_controller_test.go

```diff
@@ -194,6 +194,7 @@ func TestAdmittedNotReadyWorkload(t *testing.T) {
 
 func TestSyncCheckStates(t *testing.T) {
 	now := metav1.NewTime(time.Now())
+	fakeClock := testingclock.NewFakeClock(time.Now())
```

Comment:
```suggestion
	now := time.Now()
	fakeClock := testingclock.NewFakeClock(now)
```

Could you use the predefined current time so that we can keep using the same time everywhere?
And then, we can declare `metav1.NewTime(now)` in each LastTransitionTime.

Reply:
@tenzen-y so we don't need to call `DeepCopy` anymore ?

Reply:
I've made a PR, please check

Reply:
Yes, that's right.
Thank you!

/lgtm
/approve